api/src/main/resources/db/migration/V2__CreateMemeTable.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE memes (
+    ID SERIAL NOT NULL PRIMARY KEY,
+    memeName TEXT UNIQUE,
```

Comment:
Is it better to use TEXT or do we want to use VARCHAR and limit the number of characters?

Reply:
Since we're using Postgres, there's not a storage or performance implication to this.  In another DBMS, there is.  That said, this field having, effectively, unlimited characters doesn't make sense.  It should likely be a `VARCHAR(200)` or so.